internal/harnesses/docker/docker.go

```diff
@@ -46,6 +59,12 @@ func New(id string, cli *provider.DockerClient, opts ...Option) (types.Harness,
 		})
 	}
 
+	managedVolumes = append(managedVolumes, mount.Mount{
+		Type:   mount.TypeVolume,
+		Target: "/root/.docker",
```

Comment:
I don't love how these are passed around as strings, but concede given we're basically hardcoding the runtime user as `root`.

in the future when we need to support alternative users we should probably clean this up

Reply:
How would you prefer this done? I can refactor it in a subsequent PR.